_posts/2022-04-11-changelog-168.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #168"
+date: 2022-04-11 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8647] Collapse macro expansion steps into single one.
+It's not only fixes UI blinking during macro expansion process but also significantly speed up macro expansion itself
+
+* [#8689] Provide file links in terminal for `dbg!` macro output
+
+  {% include gif-img.html path="/assets/posts/changelog-168/dbg_output_file_links" w="700px" %}
+
+## Fixes
+
+* [#8674] The `Generate constructor` action now places the cursor after the generated constructor (by [@Kobzol])
+
+* [#8671] Escape keyword-like identifiers like `type` by `Import` quick-fix
+
+* [#8631] Properly evaluate build scripts even if a crate doesn't contain binary or library targets.
```

Comment:
Not sure, but as I understand, "package" should be used here instead of "crate" ([in terms of Cargo](https://doc.rust-lang.org/cargo/reference/cargo-targets.html))

```suggestion
* [#8631] Properly evaluate build scripts even if a package does not have any binary or library targets.
```